bootstrap/bootstrap.py

```diff
@@ -113,35 +122,64 @@ def ingest():
     data_for_output_mixin = dictor(cfg, OUTPUT_MIXIN_DATA, checknone=True)
 
     try:
-        tenant_id = get_tenant_id(tenant_id_url, headers)
-        class_id = get_class_id(create_class_url, headers, input_class_title, data_for_class)
-        input_mixin_id = get_mixin_id(create_mixin_url, headers, input_mixin_title, data_for_mixin, class_id,
+
+        tenant_id = get_tenant_id(tenant_id_url, copy.deepcopy(headers_for_ingestion))
+
+        class_id = get_class_id(create_class_url, copy.deepcopy(headers_for_ingestion), input_class_title, data_for_class)
+
+        input_mixin_id = get_mixin_id(create_mixin_url, copy.deepcopy(headers_for_ingestion), input_mixin_title, data_for_mixin, class_id,
```

Comment:
Not sure about Python optimizations but how expensive is this `copy.deepcopy(headers_for_ingestion)`?  Should we call this once and reuse the storage value for it?